engine/rpcserver.go

```diff
@@ -898,6 +904,8 @@ func (s *RPCServer) CancelOrder(ctx context.Context, r *gctrpc.CancelOrderReques
 		ID:            r.OrderId,
 		Side:          order.Side(r.Side),
 		WalletAddress: r.WalletAddress,
+		Pair : currency.NewPairFromStrings(r.Pair.Base,r.Pair.Quote),
+
 	})
 
 	return &gctrpc.CancelOrderResponse{}, err
```

Comment:
if you feel comfortable with protobuf/gRPC you could also update the CancelOrderResponse{} message here to pass back a confirmation but its kind of out of scope of the original fix and I am happy to open up a PR for it at a later time

Reply:
Yes you can , let's fix them as quickly as possible

Reply:
We can do this in a separate PR